### DISCUSSION TITLE
not fully displayed

### DISCUSSION THREAD

**Discussion by AjianNie** [Anze/KeyCluCask#72] [Q&A]
intel macOS13 Version 0.26 (22630)

I turned on the following two options in Visibility: Show user-hidden menu items; Show inactive menu items
There are so many shortcut keys in Xcode that keyclu cannot display all the content and cannot scroll down.
![截图2024-04-27 11 16 14](https://github.com/Anze/KeyCluCask/assets/108182750/bc6aaf6e-c7bc-4aeb-b09b-d007377a80cf)

Then I tried adjusting the height and width of the panel to the maximum, but it still didn't solve the problem.
![截图2024-04-27 11 40 11](https://github.com/Anze/KeyCluCask/assets/108182750/1125a68c-7abe-46fb-8907-92f5a3ef97cd)

**Comment by Anze** [comment]
Hey [AjianNie](https://github.com/AjianNie),

It looks like you've enabled the "Show all menu items" option, which might impact performance and cause scrolling to lag. This option, along with others you've enabled, isn't deemed stable enough. That's why it's categorized under "Experimental Features". I suggest turning off these options and retesting the scrolling to see if it improves.
Just curious what monitor you have, interesting resolution and DPI.

**Reply by AjianNie** [reply]
Thank you, the problem is solved🥳
And  I modified the resolution of the 13-inch display on the macbook to 1440x900
